docs/draft/mentions.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Mentions
+nav_order: 2
+has_children: true
+permalink: /specs/draft
+---
+
+# Mentions
+
+## Usage
+
+To mention a user, one would type a message containing `@0xpk` where `pk` is the public key of the user to be mentioned.
+
+## Implementation in status-go
+
+`status-go` parses the markdown in the messages and detects mentions that follow the pattern described above. This is done in [`parser.inline.go`](https://github.com/status-im/markdown/blob/b9fe921681227b1dace4b56364e15edb3b698308/parser/inline.go#L68). When mentions are found they are added as an element to the message that is passed to the client with the following format:
+
+```
+{:type "mention" :literal "0x......"}
+```
+
+The `status-react` client uses specific styling for the following elements: `code`, `emph`, `strong`, `link`, `status-tag` and `mention`.
+`mentions` will be displayed in a color that sets it apart from regular text and tapping it will open the chat with the mentioned user.
+
```

Comment:
## Copyright

Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).